specification/iotoperations/resource-manager/Microsoft.IoTOperations/preview/2024-08-15-preview/iotoperations.json

```diff
@@ -5668,7 +5668,7 @@
       "enum": [
         "SystemAssignedManagedIdentity",
         "UserAssignedManagedIdentity",
-        "Sasl",
+        "ServiceAccountToken",
```

Comment:
>ServiceAccountToken [](http://example.com/codeflow?start=9&length=19)

looks like this API version is already shipped. 

Will this removal of enum "Sasl" not break existing users who are dependent on it ?

Reply:
It wouldn't impact our customers because our customers aren't using this version yet. It's currently only being used internally.

We've gotten signoff from our PMs and we originally got breaking change signoff from  @mikekistler 

cc @chgennar @RamIoTMalhotra